openquake/hazardlib/gsim/mgmpe/modifiable_gmpe.py

```diff
@@ -49,6 +54,21 @@
                  IMC.RotD50: [1.009, 0, 1, 1, 0, 1]}
 
 
+def sigma_model_alatik2015(self, ctx, imt, ergodic, tau_model, phi_ss_coetab,
+                           tau_coetab):
+    """
```

Comment:
Please remove the docstring if you are going to fill it

Reply:
OK I added a doc string